### DISCUSSION TITLE
Why is the CLI installable?

### DISCUSSION THREAD

**Discussion by FelixKLG** [SeaQL/sea-orm#2120] [Q&A]
Just a general question as to why SeaORM requires you install the CLI globally instead of say going a similar route to prisma_client_rust where you add it to your cargo workspace, I am personally quite a fan of the PCR method as I could potentially supply a custom configuration as I personally have a disliking towards .env files.
I also find it better for version management and per-project installs, I also dislike having all my per-project tools in my global system.

Just curious is there's ever a plan to change how the sea cli works?

**Comment by tyt2y3** [comment]
You can bundle a local cli into your project, as discussed in https://github.com/SeaQL/sea-orm/discussions/1889

**Reply by Expurple** [reply]
Perhaps, we should mention this method in the documentation?
